Windows/Debugger/CtrlDisAsmView.cpp

```diff
@@ -1352,12 +1400,175 @@ void CtrlDisAsmView::disassembleToFile() {
 	}
 }
 
+
+// The following member has been modified to accomodate "nicer-looking" output (according to my needs). Assumes maximum length of an instruction name is 9 or 10 chars.
+// What's the maximum length the params string could reasonably be?
```

Comment:
I still think this should not be modified, you may add a new function like `getOpcodeTextFormatted` or something so it doesn't affect other part of the code where this is used.

Also I think all this code can be done like this (untested tho'):

```
DisassemblyLineInfo line;
address = manager.getStartAddress(address);
manager.getLine(address,displaySymbols,line);
snprintf(dest, bufsize, "%10s\t%32s", line.name.c_str(), trimString(line.params).c_str());
```

Reply:
That's a good call actually. I'll mess with that later. As for your code, well i could just steal it wholesale and test but now i feel like i should have figured it out myself XD